### ISSUE TITLE
VolatilityPeriod and strategy

### ISSUE THREAD

**Issue by nspassov** [butor/blackbird#379]
It seems some exchanges diverge often but for short periods of time. In the configuration there is the `VolatilityPeriod` parameter which is set at 600 (iterations?). Wouldn't it make sense to decrease this number in order to participate even for short periods of time (and more frequently) when there is enough price difference?

**Comment by CCKRK** [comment]
I could be wrong but I believe the volatility is not used for anything. The volatility period is just the number of iterations volatility of the spread would be calculated on if you enable it. Blackbird is probably not executing the trades you see due to the trailing system, see #283 and #272.

**Comment by butor** [comment]
That's correct @CCKRK. The volatility is only displayed for reference, but it is not used for the moment (see #6)
